packages/material-ui-styles/src/ThemeProvider/ThemeProvider.js

```diff
@@ -61,7 +64,25 @@ function ThemeProvider(props) {
     return output;
   }, [localTheme, outerTheme]);
 
-  return <ThemeContext.Provider value={theme}>{children}</ThemeContext.Provider>;
+  const cacheLtr = createCache({
```

Comment:
This is for the rtl support

Reply:
Is it ok that `cache` will be recreated on each function call?

They use it as a singleton in the example https://github.com/emotion-js/emotion/tree/master/packages/cache#createcache

Reply:
Good point! Fixed.